counsel.el

```diff
@@ -3066,7 +3066,8 @@ Works for `counsel-git-grep', `counsel-ag', etc."
                    (all-args (append
                               (when (car command-args)
                                 (split-string (car command-args)))
-                              (counsel--ag-extra-switches regex)
+                              (when-let ((extra-switches (counsel--ag-extra-switches regex)))
```

Comment:
There's no `when-let` in the minimum supported Emacs version of 24.5.

Reply:
@basil-conto thanks for the feedback. I've adjusted the PR to use the existing convention of using when & let. Let me know if this is what you had in mind, if not I'm open to suggestions (I am a novice elisper)

Reply:
Yep that's fine @rymndhng, thanks.